### ISSUE TITLE
[Bug] Togglz Console: Toggling feature in custom feature group redirects to "All Features"

### ISSUE THREAD

**Issue by eancuta** [togglz/togglz#264]
## Expected Behavior
Toggling a feature in the console/admin UI should keep you in your selected group.

## Current Behavior
Toggling a feature in a feature group will disable/enable the feature, then auto-select the all features group in the console.

## Possible Solution
Remove the auto-selection

## Context
I am new to Togglz and coding. I am currently adding many toggles to an existing project. Sometimes, I have to enable multiple toggles which I hold in a "Test" feature group. It can be annoying having to reclick on the "Test" feature group.

**Comment by chkal** [comment]
I agree that the current behavior isn't very user-friendly. I was thinking about implementing the console from the scratch with React at some point. In this case I will definitely build it in a way that preserves the current state of the toggle list page.

**Comment by jorgheymans** [comment]
This for me is very annoying as we have a large number of features that are duplicated in several feature groups, if you don't pay attention that the UI dropped you back into the 'all features' group you end up toggling features that you did not mean to.

Could a cookie based solution that remembers the last active feature group not be a quick win here ?
